csharp/tests/Facebook.Yoga.Universal.Tests/Facebook.Yoga.Universal.Tests.csproj

```diff
@@ -0,0 +1,146 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd definitely like to do a follow-up PR to make the unit tests unit NUnit 3.x package, and that nuget package *does* have UWP support. I appreciate keeping this PR as narrow as possible given the scope.